permissions/plugin-secone-sca-sast-security-scanner.yml

```diff
@@ -8,3 +8,5 @@ developers:
 - "saurabhthatte_sec1"
 issues:
   - jira: 29523
+cd:
+  enabled: true
```

Comment:
your scm.tag is wrong in the developerConnection, it should be this:
https://github.com/jenkinsci/archetypes/blob/master/empty-plugin/src/main/resources/archetype-resources/pom.xml#L29

the rest looks good

Reply:
as suggested updated the pom.